src/libcore/iter/iterator.rs

```diff
@@ -258,6 +258,32 @@ pub trait Iterator {
         None
     }
 
+    /// Returns an iterator adaptor that goes over the original iterator,
+    /// skipping `step` elements at a time.
+    ///
+    /// **Panics** if `step == 0`.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// let a = [0, 1, 2, 3, 4, 5];
+    ///
+    /// let mut iter = a.iter().every_nth(2);
+    ///
+    /// assert_eq!(iter.next(), Some(&0));
+    /// assert_eq!(iter.next(), Some(&2));
+    /// assert_eq!(iter.next(), Some(&4));
+    /// assert_eq!(iter.next(), None);
+    /// ```
+    #[inline]
+    #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
You'll want to fix this attribute. It should be given its own feature name and it should be unstable.